src/test/java/guitests/UITest.java

```diff
@@ -337,4 +337,12 @@ public GuiTest type(String text) {
         return this;
     }
 
+    /**
+     * NOTE: please turn off NumLock as it will interfere with this method
+     *
+     * Used to select the whole filter text so that it can be replaced
+     */
+    public void selectAll() {
```

Comment:
Is it possible to verify NumLock is turned off and throw an exception if it is on?